File: Java/topflytechCodec/src/com/topflytech/codec/entities/WifiMessage.java
package com.topflytech.codec.entities;

import java.util.Date;

/**
 * Created by admin on 2021/3/11.
 */
public class WifiMessage  extends Message{
    private Date date;
    private String selfMac;
    private String ap1Mac;
    private int ap1RSSI;
    private String ap2Mac;
    private int ap2RSSI;
    private String ap3Mac;
    private int ap3RSSI;


    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public String getSelfMac() {
        return selfMac;
    }

    public void setSelfMac(String selfMac) {
        this.selfMac = selfMac;
    }

    public String getAp1Mac() {
        return ap1Mac;
    }

    public void setAp1Mac(String ap1Mac) {
        this.ap1Mac = ap1Mac;
    }

    public int getAp1RSSI() {
        return ap1RSSI;
    }

    public void setAp1RSSI(int ap1RSSI) {
        this.ap1RSSI = ap1RSSI;
    }

    public String getAp2Mac() {
        return ap2Mac;
    }

    public void setAp2Mac(String ap2Mac) {
        this.ap2Mac = ap2Mac;
    }

    public int getAp2RSSI() {
        return ap2RSSI;
    }

    public void setAp2RSSI(int ap2RSSI) {
        this.ap2RSSI = ap2RSSI;
    }

    public String getAp3Mac() {
        return ap3Mac;
    }

    public void setAp3Mac(String ap3Mac) {
        this.ap3Mac = ap3Mac;
    }

    public int getAp3RSSI() {
        return ap3RSSI;
    }

    public void setAp3RSSI(int ap3RSSI) {
        this.ap3RSSI = ap3RSSI;
    }


}

File: C#/TopflytechCodec/TopflytechCodec/Entities/WifiMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    public class WifiMessage : Message
    {
        private DateTime date;
        public DateTime Date
        {
            get { return date; }
            set { date = value; }
        }
        private String selfMac;
        public String SelfMac
        {
            get { return selfMac; }
            set { selfMac = value; }
        }
        private String ap1Mac;
        public String Ap1Mac
        {
            get { return ap1Mac; }
            set { ap1Mac = value; }
        }
        private int ap1RSSI;
        public int Ap1RSSI
        {
            get { return ap1RSSI; }
            set { ap1RSSI = value; }
        }
        private String ap2Mac;
        public String Ap2Mac
        {
            get { return ap2Mac; }
            set { ap2Mac = value; }
        }
        private int ap2RSSI;
        public int Ap2RSSI
        {
            get { return ap2RSSI; }
            set { ap2RSSI = value; }
        }
        private String ap3Mac;
        public String Ap3Mac
        {
            get { return ap3Mac; }
            set { ap3Mac = value; }
        }
        private int ap3RSSI;
        public int Ap3RSSI
        {
            get { return ap3RSSI; }
            set { ap3RSSI = value; }
        }



    }
}
